docs/docs/cmd/spo/tenant/tenant-site-membership-list.mdx

```diff
@@ -0,0 +1,93 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# spo tenant site membership list
+
+Retrieve information about default site groups' membership
+
+## Usage
+
+```sh
+m365 spo tenant site membership list [options]
+```
+
+## Options
+
+```md definition-list
+`-u, --siteUrl <siteUrl>`
+: The URL of the site.
+
+`-r, --role [role]`
+: Filter the results to include only users with the specified roles: 'Owner', 'Member', or 'Visitor'.
+
+```
+
+<Global />
+
+## Remarks
+
+:::info
+
+To execute this command successfully, ensure you are logged in to the CLI for M365 with an account that has SharePoint admin permissions. For other scenarios, refer to the `spo web get --withGroups` and `spo group member list` commands.
+
+:::
+
+## Examples
+
+Retrieves information about default site groups' owners, members, and visitors of the site.
+
+```sh
+m365 spo tenant site membership list --siteUrl https://contoso.sharepoint.com
+```
+
+Retrieves information about site owners.
+
+```sh
+m365 spo tenant site membership list --siteUrl https://contoso.sharepoint.com --role Owner
+```
+
+## Response
+
+<Tabs>
+  <TabItem value="JSON">
```

Comment:
I'm a bit struggling with the fact that this is a `list` command, that doesn't output an array, but a single object. What is your take on this @pnp/cli-for-microsoft-365-maintainers?

![image](https://github.com/user-attachments/assets/686131d0-7613-478c-81d0-d36820f1a53c)

Reply:
Anyway, we have a few exceptions like `spo settings list`, `planner settings list`, `planner task checklistitem list`, ...

Reply:
Ok, so I left it as it is